content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md

```diff
@@ -51,3 +51,10 @@ bash-completion sources all completion scripts in `/etc/bash_completion.d`.
 {{< /note >}}
 
 Both approaches are equivalent. After reloading your shell, kubectl autocompletion should be working.
+
+To enable autocompletion in current session of shell, source the .bashrc file:
```

Comment:
Does this work if the user picked the “System” option, editing `/etc/bash_completion.d/kubectl`?